app/controllers/your-psp/your-psp-tasks.service.test.js

```diff
@@ -204,5 +204,174 @@ describe('Stripe task list', () => {
       const taskList = getStripeTaskList(activeCredential, account)
       expect(stripeTaskListIsComplete(taskList)).to.equal(false)
     })
+
+    it('should correctly calculate when zero tasks are complete ', () => {
```

Comment:
Would be good to nest these tests into a separate describe as these are for a specific functionality `stripeTaskListNumberOftasksComplete`.